rockspec.in

```diff
@@ -9,7 +9,7 @@ description = {
 	 directly from Lua.
    ]],
    license = 'MIT/X11',
-   homepage = 'http://github.com/pavouk/lgi'
+   homepage = 'https://github.com/pavouk/lgi'
```

Comment:
`github.com/pavouk/lgi` reads now `github.com/lgi-devs/lgi/`.

This applies to both `https://` and `git://`